package.json

```diff
@@ -12,29 +12,29 @@
   "version": "1.7.4",
   "license": "GPL-3.0",
   "description": "A powerful Markdown editor for researchers.",
-  "main": "./source/main.js",
+  "main": ".webpack/main",
   "scripts": {
-    "postinstall": "electron-builder install-app-deps && yarn less && yarn handlebars && yarn reveal:build && yarn wp:prod",
-    "start": "electron .",
-    "build:quick": "electron-builder --dir",
-    "release:this": "electron-builder",
-    "release:mac": "electron-builder --mac --publish \"never\"",
-    "release:win": "electron-builder --win --x64 --ia32 --publish \"never\"",
-    "release:win-arm": "electron-builder --win --ia32 --arm64 --publish \"never\"",
-    "release:linux": "electron-builder --linux --publish \"never\"",
-    "less": "gulp less",
-    "handlebars": "gulp handlebars",
+    "postinstall": "electron-builder install-app-deps",
+    "start": "electron-forge start",
+    "package": "electron-forge package",
+    "package:mac": "electron-forge package --platform=darwin",
+    "package:win": "electron-forge package --platform=win32 --arch=ia32,x64",
+    "package:win-arm": "electron-forge package --platform=win32 --arch=arm64",
+    "package:linux": "electron-forge package --platform=linux",
+    "release": "yarn package && electron-builder",
+    "release:mac": "yarn package:mac && electron-builder --mac --publish \"never\" --prepackaged out/TODO",
+    "release:win": "yarn package:win && electron-builder --win --x64 --ia32 --publish \"never\" --prepackaged out/zettlr-win32-x64",
+    "release:win-arm": "yarn package:win-arm && electron-builder --win --ia32 --arm64 --publish \"never\" --prepackaged out/TODO",
+    "release:linux": "yarn package:linux && electron-builder --linux --publish \"never\" --prepackaged out/TODO",
     "lang:refresh": "node scripts/refresh-language.js",
     "csl:refresh": "node scripts/update-csl.js",
     "reveal:build": "node scripts/update-reveal.js",
```

Comment:
Memo to ourselves: Adapt the available commands in the Readme as well as the documentation